care/facility/models/patient.py

```diff
@@ -733,3 +733,22 @@ def get_related_consultation(self):
         # and hence the permission mixin will fail if edit/object_read permissions are checked (although not used as of now)
         # Remove once patient notes is made consultation specific.
         return self
+
+
+class PatientNotesEdit(models.Model):
+    patient_note = models.ForeignKey(
+        PatientNotes,
+        on_delete=models.CASCADE,
+        null=False,
+        blank=False,
+        related_name="edits",
+    )
+    edited_on = models.DateTimeField(auto_now_add=True)
```

Comment:
```suggestion
    edited_date = models.DateTimeField(auto_now_add=True)
```
The base model uses _date for created and modified date, so let's use that here as well